<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>moreInfo</class>
 <widget class="QMainWindow" name="moreInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>701</width>
    <height>381</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/images/images/ALogo.png</normaloff>:/images/images/ALogo.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" rowspan="3">
     <layout class="QVBoxLayout" name="vlProcesses"/>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="vlRAM"/>
    </item>
    <item row="1" column="1">
     <layout class="QVBoxLayout" name="vlCPU"/>
    </item>
    <item row="2" column="1" rowspan="2">
     <layout class="QVBoxLayout" name="vlNetStat"/>
    </item>
    <item row="3" column="0">
     <layout class="QVBoxLayout" name="vlHDD"/>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
